spec/lib/dor/release/member_services_spec.rb

```diff
@@ -0,0 +1,31 @@
+# frozen_string_literal: true
+
+require 'spec_helper'
+
+RSpec.describe Dor::Release::MemberService do
+  let(:member_service) { described_class.new(druid: 'druid:oo000oo0001') }
+  let(:member) { Struct.new(:externalIdentifier, :type, keyword_init: true) }
+  let(:item_member) { member.new(externalIdentifier: 'druid:xx828xx3282', type: ['item']) }
+  let(:collection_member) { member.new(externalIdentifier: 'druid:xx222xx3282', type: ['collection']) }
```

Comment:
and how about `instance_double(Cocina::Models::Collection, ...)` here?

Reply:
good idea, changed